doc/classes/ShortCut.xml

```diff
@@ -16,7 +16,7 @@
 			<return type="String">
 			</return>
 			<description>
-				Returns the Shortcut's [InputEvent] as a [String].
+				Returns the shortcut's [InputEvent] as a [String].
```

Comment:
I would recommend capitalizing this as ShortCut since I believe that's referring to the class name.

Reply:
There are a lot of places that use overcapitalization. Member descriptions in the classref are written referring to an object's properties, not the class.  "Hide the sprite's texture" versus "Hide the Sprite's Texture".

Reply:
I concur, capitalization should be reserved for when an explicit reference to a class (and thus `[Class]` or `[code]Class`[/code]) is needed.